verifier/src/lib.rs

```diff
@@ -1,255 +1,39 @@
-use ark_bn254::{Fq, G1Affine};
-use ark_ff::PrimeField;
-use ark_serialize::CanonicalSerialize;
-use borsh::BorshSerialize;
+//! # Verifier
+//!
+//! This crate contains utilities for verifying SP1 Groth16 proofs on Solana.
+//!
+//! # Example
+//! ```
```

Comment:
Does this need to have the tag like ```rust on it?

Reply:
to run as a doctest, no, since it's in a rust file